### ISSUE TITLE
Backups with checkpoints --> inaccurate stats?

### ISSUE THREAD

**Issue by geckolinux** [borgbase/vorta#484]
Hi there, running Vorta 0.6.26 on openSUSE Tumbleweed.

In addition to BorgBackup, I'm also backing up my `~/` with a separate profile to a NAS. I finished the first complete backup in multiple steps (testing different compression levels and letting it run for 5 minutes to check transfer speed, then went to bed and resumed the next morning until it finished). Vorta now reports this:

![Screenshot_20200601_200341](https://user-images.githubusercontent.com/2692138/83468928-dca84200-a443-11ea-83d6-e6f310b7bd5e.png)

I'm not sure if I'm not understanding something, or maybe it's a Borg bug and not specific to Vorta, but the original data that was backed up was only about 79 GiB, so the "Original size" stat feels wrong. I wonder if it's counting the same data that was going to be backed up for each of the checkpoints?

![Screenshot_20200601_200548](https://user-images.githubusercontent.com/2692138/83469056-390b6180-a444-11ea-8ceb-c657f15a0d1c.png)

Is it supposed to show those checkpoints? After the successful complete backup shouldn't the checkpoints be removed? And can I manually delete the checkpoints and still keep all the data in the final snapshot?

Thanks a lot.

**Comment by m3nu** [comment]
"Original Size" sums up *all* the files in *all* the archives. Checkpoints count as archives. So it would be roughly `4*79gb~=300gb`. After that you see deduplication and compression kicking in.

This is roughly how Borg reports the space usage. There is an argument to display the size of a single archive instead of sum of all archives. But this breaks down if I add more folders later. So not sure this should be changed. We had a similar question a few days ago. https://github.com/borgbase/vorta/issues/472

**Comment by geckolinux** [comment]
That makes sense, maybe it's just a question of semantics. I guess I would have expected a stat with the size of all the data available at that point in time, i.e. the same figure I get when mounting the snapshot and checking the mount directory's recursive size with a file manager.

And can I delete the checkpoints now and still have a complete final snapshot archive? I understand the concept in Borg that every snapshot is effectively like a full backup with no dependencies on the other ones, but the `.checkpoint` suffix makes me wonder if those are a special case.

**Comment by m3nu** [comment]
Short answer: yes

Long answer: https://borgbackup.readthedocs.io/en/stable/faq.html#if-a-backup-stops-mid-way-does-the-already-backed-up-data-stay-there

**Comment by hutson** [comment]
@m3nu I ended up here after I, too, was confused by the behavior of the `Original Size` value on the Repository tab. I had expected it to correlate with either the total size of my files on disk (On my laptop), or the total size of the repository on BorgBackup. It was not until I read this issue, along with #430 and #214 that I learned to expect `Original Size` to represent the sum of all archives along with any data referenced by an archive (Even if the original archive was pruned).

My opinion, but I'm not sure `Original Size` is meaningful to me as the end-user. What I cared about was the total size of files on my system before running the first backup (because that represented the maximum amount of disk space I might consume on the remote repository), and the total size of versioned files stored on the remote repository (Because that is what I pay for).
